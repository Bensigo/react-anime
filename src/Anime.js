// @flow

import React from 'react';
import anime from 'animejs';

class Anime extends React.Component {
  constructor () {
    super()
  }

  componentDidMount () {

  }

  componentWillUnmount () {

  }

  render () {
    return null; 
  }
}

export default Anime;
