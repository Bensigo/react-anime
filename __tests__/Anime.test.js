import React from 'react';
import renderer from 'react-test-renderer';
import Anime from './../src/Anime.js';

// refular function / expression test / unit test
test('Anime', () => {
  expect(1).toBe(1);
})

//snapshot test
it('Anime Snapshot', () => {
  const tree = renderer.create(
    <Anime />
  ).toJSON();
  expect(tree).toMatchSnapshot();
})
