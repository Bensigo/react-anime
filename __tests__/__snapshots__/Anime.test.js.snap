exports[`test Anime Snapshot 1`] = `null`;
